freeipa/src/main/java/com/sequenceiq/freeipa/kerberosmgmt/v1/KerberosMgmtV1Service.java

```diff
@@ -64,4 +111,114 @@ public void deleteHost(HostRequest request) {
         //TODO  Will be implemented CDPSDX-515
         return;
     }
+
+    private Stack getFreeIpaStack(String envCrn) {
+        FreeIpaClient freeIpaClient = null;
+        String failure;
+        String accountId = crnService.getCurrentAccountId();
```

Comment:
we usually get accountId early, like in the controller so it would fail early if it's missing